parser/sql/dialect/doris/pom.xml

```diff
@@ -36,5 +36,11 @@
             <artifactId>shardingsphere-infra-database-doris</artifactId>
             <version>${project.version}</version>
         </dependency>
+        
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <scope>test</scope>
```

Comment:
Similar to the situation in `shardingsphere-standalone-mode-repository-jdbc`, it is already in `test` scope.

<img width="530" alt="image" src="https://github.com/user-attachments/assets/c36ab618-6891-4aff-aef0-8717a7049185">

Reply:
- See https://github.com/apache/shardingsphere/pull/32389#discussion_r1703490403 . After I removed the declaration of `ch.qos.logback:logback-classic` here, I didn't see any logback dependencies. Without explicit declaration, the `parser/sql/dialect/doris/src/test/resources/logback-test.xml` of this module will be useless. Where does this picture come from?
- ![image](https://github.com/user-attachments/assets/f1eaf8f1-2036-45aa-9b3c-26c5dd4d8788)

Reply:
<img width="1643" alt="image" src="https://github.com/user-attachments/assets/94e81ea4-25d9-4c11-b351-6d8a6de02136">

Reply:
It looks like you are looking at the dependencies of the master branch. On the master branch, logback in test scope is pulled into all modules, as I pointed out in https://github.com/apache/shardingsphere/pull/32389#discussion_r1703174856 .

Reply:
@linghengqian Yes, I found the difference, in this PR you removed logback from the root pom and added it to specific modules, just to make logback not appear in the test scope downstream?

Reply:
Yes. Because if we do not exclude the test scope logback globally, after a module introduces Hiveserver2 JDBC Driver, multiple slf4j adapters including Reload4j, Log4j2, Logback, Jakarta Commons Logging, etc. will still appear in the classpath of the unit test at the same time, which violates https://www.slf4j.org/codes.html#multiple_bindings . I think a better solution is to explicitly declare Logback in the module that needs to use Logback. What do you think?

Reply:
I agree with your point. How do you know that these modules need to depend on logback? (There are quite a lot of them)

Reply:
Yes, the Java API may be introduced by logging rule, and the others are all in logback.xml

Thanks for your clarification.

Reply:
> Yes. Because if we do not exclude the test scope logback globally, after a module introduces Hiveserver2 JDBC Driver, multiple slf4j adapters including Reload4j, Log4j2, Logback, Jakarta Commons Logging, etc. will still appear in the classpath of the unit test at the same time, which violates https://www.slf4j.org/codes.html#multiple_bindings . I think a better solution is to explicitly declare Logback in the module that needs to use Logback. What do you think?

If we use <exclusion> to exclude incompatible dependencies when introducing new dependency, such as log4j introduced by hive, to ensure the simplicity of ShardingSphere itself, is this possible?

Reply:
- This is actually what I asked you at https://github.com/apache/shardingsphere/pull/32389#discussion_r1703174856 . We have 2 options, one is to keep the test scope Logback globally, the other is to remove the test scope Logback globally. We can of course limit unit tests to use only Logback. If you think the test scope Log should be kept globally, another commit will be pushed.
```xml
        <dependency>
            <groupId>org.apache.hive</groupId>
            <artifactId>hive-jdbc</artifactId>
            <version>4.0.0</version>
        </dependency>
        <!-- See https://issues.apache.org/jira/browse/HIVE-28308 -->
        <dependency>
            <groupId>org.apache.hive</groupId>
            <artifactId>hive-service</artifactId>
            <version>4.0.0</version>
            <exclusions>
                <!-- See https://issues.apache.org/jira/browse/HIVE-28429 -->
                <exclusion>
                    <groupId>org.apache.hadoop</groupId>
                    <artifactId>hadoop-client-api</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>org.apache.logging.log4j</groupId>
                    <artifactId>log4j-api</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>org.apache.logging.log4j</groupId>
                    <artifactId>log4j-slf4j-impl</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>org.slf4j</groupId>
                    <artifactId>slf4j-log4j12</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
        <dependency>
            <groupId>org.apache.hadoop</groupId>
            <artifactId>hadoop-client-api</artifactId>
            <version>3.3.6</version>
        </dependency>
```

Reply:
> * This is actually what I asked you at [Block `shardingsphere-logging-core` from passing `logback-classic` dependencies to downstream modules #32389 (comment)](https://github.com/apache/shardingsphere/pull/32389#discussion_r1703174856) . We have 2 options, one is to keep the test scope Logback globally, the other is to remove the test scope Logback globally. We can of course limit unit tests to use only Logback. If you think the test scope Log should be kept globally, another commit will be pushed.
> 

Sorry, I misunderstood your intention. I thought we were going to remove the logback dependency introduced by the logging rule.

For scenarios like hive, the \<exclusion> method is more acceptable, and ShardingSphere developers can still focus on logback.

Reply:
- Done.

Reply:
@linghengqian Thank you!